perf/.eslintrc

```diff
@@ -2,5 +2,7 @@
   rules: {
     no-console: off,
     no-process-exit: off,
+    no-unused-vars: off,
+    no-empty: off
```

Comment:
Why though?

Reply:
For the empty loops, perhaps just have an assertion in there (even if just `assert(quad)`.